.env

```
@@ -0,0 +1,3 @@
+OPENAI_API_KEY=lmstudio
+OPENAI_BASE_URL=http://localhost:1234/v1/
+OPENAI_MODEL=glm-4.5-air
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)